src/main/asciidoc/net.adoc

```diff
@@ -770,3 +770,27 @@ Here's an example:
 
 The DNS resolution is always done on the proxy server, to achieve the functionality of a SOCKS4 client, it is necessary
 to resolve the DNS address locally.
+
+
+=== Using HA PROXY protocol
+
+https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[HA PROXY protocol] provides a convenient way to safely transport connection
+information such as a client's address across multiple layers of NAT or TCP
+proxies.
+
+HA PROXY protocol can be enabled by setting the option {@link io.vertx.core.http.NetServerOptions#useProxyProtocol(boolean)}
+and adding the following dependency in your classpath:
```

Comment:
_to_ your classpath ?

Reply:
=== Native Linux Transport

You need to add the following dependency in your classpath:

Reply:
that might be wrong too :-)

Reply:
it seems that using "in" is correct

Reply:
sorry for noise